ui/yarn.lock

```diff
@@ -2233,17 +2233,10 @@
   resolved "https://registry.yarnpkg.com/@types/range-parser/-/range-parser-1.2.4.tgz#cd667bcfdd025213aafb7ca5915a932590acdcdc"
   integrity sha512-EEhsLsD6UsDM1yFhAvy0Cjr6VwmpMWqFBCb9w07wVugF7w9nfajxLuVmngTIpgS6svCnm6Vaw+MZhoDCKnOfsw==
 
-"@types/react-dom@*":
-  version "18.2.4"
```

Comment:
I'm surprised that this wasn't already causing problems.  It's important to keep all React types on a 17.x version, or you see weird type errors like "ReactNode does not match ReactNode"